tests/EvmLookupServer/Test_EvmLookupServer.cpp

```diff
@@ -173,7 +171,8 @@ BOOST_AUTO_TEST_SUITE(BOOST_TEST_MODULE)
  */
 class GetEthCallEvmClientMock : public EvmClient {
  public:
-  virtual bool OpenServer(uint32_t /*force = false*/) override { return true; };
+  //  virtual bool OpenServer(uint32_t /*force = false*/) override { return
```

Comment:
Please remove commented out methods.  We have git history for that.

Reply:
Not my usual style, a person no longer with us asked me to start doing that, I shall now throw away that request.

Fixed